From: David Milburn <dmilburn@redhat.com>
Date: Thu, 18 Nov 2010 20:40:48 -0500
Subject: [ata] ata_piix: IDE Mode SATA for Intel Patsburg Devices
Message-id: <1290112856-1796-2-git-send-email-dmilburn@redhat.com>
Patchwork-id: 29505
O-Subject: [RHEL5.6 PATCH 1/9] BZ 570868 ata_piix: IDE Mode SATA patch for Intel
	Patsburg DeviceIDs
Bugzilla: 570868
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Michal Schmidt <mschmidt@redhat.com>

commit 238e149c7a92eb79ab9f48c171e907a5bde18333
Author: Seth Heasley <seth.heasley@intel.com>
Date:   Thu Sep 9 09:42:40 2010 -0700

    ata_piix: IDE Mode SATA patch for Intel Patsburg DeviceIDs

    This patch adds the Intel Patsburg (PCH) IDE mode SATA Controller DeviceIDs.

BZ 570868

diff --git a/drivers/ata/ata_piix.c b/drivers/ata/ata_piix.c
index 724a082..94cef77 100644
--- a/drivers/ata/ata_piix.c
+++ b/drivers/ata/ata_piix.c
@@ -294,6 +294,10 @@ static const struct pci_device_id piix_pci_tbl[] = {
 	{ 0x8086, 0x1c08, PCI_ANY_ID, PCI_ANY_ID, 0, 0, ich8_2port_sata },
 	/* SATA Controller IDE (CPT) */
 	{ 0x8086, 0x1c09, PCI_ANY_ID, PCI_ANY_ID, 0, 0, ich8_2port_sata },
+	/* SATA Controller IDE (PBG) */
+	{ 0x8086, 0x1d00, PCI_ANY_ID, PCI_ANY_ID, 0, 0, ich8_sata },
+	/* SATA Controller IDE (PBG) */
+	{ 0x8086, 0x1d08, PCI_ANY_ID, PCI_ANY_ID, 0, 0, ich8_2port_sata },
 	{ }	/* terminate list */
 };
 
